client/src/components/Blog/PostCard.jsx

```diff
@@ -0,0 +1,32 @@
+import { useContext } from "react";
+import { AuthContext } from "../../context/auth.context";
+
+export default function PostCard() {
+  const { user } = useContext(AuthContext);
+
+
+  return (
+    <div className="max-w-sm rounded overflow-hidden shadow-lg">
+    
+    <div className="px-6 py-4">
+      <div className="font-bold text-xl mb-2">Title</div>
+      <p className="text-gray-700 text-base">
+        Body
+      </p>
+    </div>
+
+    <div className="flex items-center mt-4">
+      <img className="w-10 h-10 rounded-full mr-4" src="" alt="" />
+      <div className="text-sm">
+        <p className="text-gray-900 leading-none">Created by: {user.name} </p>
```

Comment:
probably you need to add a conditional check to ensure that the user object exists before accessing its properties {user?.name}